polkadot/xcm/xcm-executor/src/traits/asset_exchange.rs

```diff
@@ -37,6 +37,13 @@ pub trait AssetExchange {
 		want: &Assets,
 		maximal: bool,
 	) -> Result<AssetsInHolding, AssetsInHolding>;
+
+	/// Handler for quoting the exchange price of two assets.
+	///
+	/// - `asset1` The first asset.
+	/// - `asset2` The second asset.
+	/// - `maximal`: If `true`, then all of `asset1` should be used.
+	fn quote_exchange_price(asset1: &Asset, asset2: &Asset, maximal: bool) -> Option<u128>;
```

Comment:
We can stop this from being a breaking change if we add a default implementation for this that just returns `None`. Is it worth it? @acatangiu

Reply:
Yeah, that sounds good.